Detectors/TRD/base/include/TRDBase/Tracklet.h

```diff
@@ -114,6 +122,10 @@ class Tracklet
   void setNHits(int nhits) { mNHits = nhits; }
   void setNHits0(int nhits) { mNHits0 = nhits; }
   void setNHits1(int nhits) { mNHits1 = nhits; }
+  void setPID(float pid) { mPID = pid; }
+  void setZ(float z) { LOG(fatal) << "Cant call setZ in Tracklet class as its a property of the detector"; }
```

Comment:
why introducing the function and then putting a log(fatal)?

Reply:
removed